### ISSUE TITLE
program to change the desktop background based on your distance from the computer

### ISSUE THREAD

**Issue by silky** [hahayes/hahayes-projects#24]
could measure distance by webcam (easy) or by sound? (impossible?)

anyway. in this way, if you had say a few shots of a scene, then you could "step foward" by moving your head closer. and look at a tree, say.

**Comment by mobeets** [comment]
yeah, this would be fun! if you could do it with sound that would probably be very bizarre indeed. you could yell to get closer.

**Comment by silky** [comment]
hahahahahaha
